sql_migration_assistant/requirements.txt

```diff
@@ -0,0 +1,6 @@
+databricks-sdk==0.30.0
+gradio==4.27.0
+pyyaml
+scipy
+mlflow[databricks]
```

Comment:
we need mlflow to get the model version from a model registered in UC, as UC makes aliases accessible rather than versions, but model serving needs versions